processing/src/main/java/io/druid/segment/DimensionHandlerUtils.java

```diff
@@ -222,10 +236,10 @@ private static ColumnCapabilities getEffectiveCapabilities(
     return strategyFactory.makeColumnSelectorStrategy(capabilities, selector);
   }
 
-  public static Long convertObjectToLong(Object valObj)
+  public static Long convertObjectToLong(@Nullable Object valObj)
```

Comment:
this is out of the scope of this PR and it will fixed by https://github.com/druid-io/druid/pull/4509/files